deploy/firesim

```diff
@@ -125,9 +126,30 @@ def managerinit(args: argparse.Namespace):
             m = local("""cp sample-backup-configs/sample_config_{}.yaml config_{}.yaml""".format(conf_file, conf_file), capture=True)
             rootLogger.debug(m)
             rootLogger.debug(m.stderr)
-            m = local("""sed -i 's/AWSUSERNAME/{}/g' config_{}.yaml""".format(get_aws_userid(), conf_file), capture=True)
-            rootLogger.debug(m)
-            rootLogger.debug(m.stderr)
+
+            if args.platform == 'f1':
+                m = local("""sed -i 's/AWSUSERNAME/{}/g' config_{}.yaml""".format(get_aws_userid(), conf_file), capture=True)
```

Comment:
I thought there was a PR somewhere that removed the need for this?

i.e. we were just going to append the ID to the bucket name automatically with an enable/disable flag.

Reply:
Thats part of this PR: https://github.com/firesim/firesim/pull/1046. I think the plan was to merge this first then revisit it. IIRC, we wanted to figure out where the recipes for the different configurations would reside (as part of the manager config files or separately) before merging it in.

Reply:
ah okay yeah, thanks